1-es-index.html

```diff
@@ -40,10 +40,10 @@ <h2>¿Necesitas ayuda?</h2>
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Jamulus es de Código Abierto</h2>
+    <h2>¡Contribuye!</h2>
     <p>
-      ¿Ideas? ¿Has encontrado un fallo? ¿Quieres contribuir? Ya que Jamulus es de código abierto, ¡nos puedes ayudar! ¡Puedes encontrar el proyecto en <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a> y el <a
-        href="https://github.com/corrados/jamulus" target="_blank" rel="noreferrer">código fuente en GitHub</a>! Jamulus se publica bajo la (<a href="http://www.gnu.org/licenses/gpl-2.0.html" target="_blank" rel="noreferrer">GPL, GNU General Public License</a>).
+      ¿Ideas? ¿Has encontrado un fallo? ¿Quieres contribuir? Ya que Jamulus es de <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">código libre y abierto</a> publicado bajo la <a href="http://www.gnu.org/licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a> ¡nos puedes ayudar!<br/>
```

Comment:
You could in fact link https://www.gnu.org/philosophy/free-sw.es.html here

Reply:
Done, cheers.

Reply:
Great! Thanks